pjmedia/include/pjmedia/conference.h

```diff
@@ -1,4 +1,4 @@
-/* 
```

Comment:
I would suggest to avoid all these whitespace modifications to make the review much easier.

If it's not possible, you can make a separate PR instead specifically for this formatting fix, so this PR will be clean.